src/Main/Main.js

```diff
@@ -0,0 +1,25 @@
+import './Main.scss'
+import burgerImage from './burgers.webp'
+import Navbar from '../Navbar/Navbar'
+import BannerBoxFoodDelivered from '../BannerBoxFoodDelivered/BannerBoxFoodDelivered'
+
+function Main() {
+  return (
+    <main className="Main">
+      <Navbar />
+      <div style={{backgroundImage: "url('" + burgerImage + "')"}} className="burger-image m-md-3">
+        <div className="row justify-content-center align-items-center h-100 m-3">
+          <div className="col-sm-12 col-md-5 semi-transparent bg-light h-50 flex-row justify-content-center align-items-center d-flex m-0">
```

Comment:
there is a point in the level of responsiveness where the text 'Order your favourite food from local restaurants, right to your door.' spills underneath the semi-transparent box,

Reply:
845px seems to be the breakpoint!

Reply:
We've changed the col-md-5 to col-md-6 - hopefully that's fixed it but we can't test it here as we aren't getting the problem. Can you test it again at your end to see if it.worked?

Reply:
875 breakpoint!